go/vt/sqlparser/keywords.go

```diff
@@ -482,6 +482,11 @@ var keywords = []keyword{
 	{"redundant", REDUNDANT},
 	{"references", REFERENCES},
 	{"regexp", REGEXP},
+	{"regexp_instr", REGEXP_INSTR},
+	{"regexp_like", REGEXP_LIKE},
+	{"regexp_replace", REGEXP_REPLACE},
+	{"regexp_substr", REGEXP_SUBSTR},
+	{"rlike", RLIKE},
```

Comment:
RLIKE was in the correct alphabetic position before

Reply:
Updated.